loadrunner_professional/assets/logs/loadrunner_professional_tests.yaml

```diff
@@ -0,0 +1,77 @@
+id: "loadrunner-professional"
+tests:
+  - sample: >+
+        {
+      		"host": "controller_host",
```

Comment:
Is this the format that is sent to datadog's intake ?

It looks like the output that the datadog api returns when logs are queried. See this [doc](https://docs.datadoghq.com/developers/integrations/log_integration/#review-and-deploy-the-integration)